gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc

```diff
@@ -77,9 +77,27 @@ to generate set of instructions that includes a command block with the registrat
 // tag::reg-command-intro[]
 To register a Flex Gateway with Anypoint Platform, you must enter the registration command and then the start command. Each command includes information specific to your Anypoint Platform instance and *must be updated before* executing. See <<Before You Begin>> for more information on how to find the information you will need.
 //end::reg-command-intro[]
+//
+// tag::k8s-connected-intro[]
+In this mode, the gateway is managed entirely through Anypoint Platform, where you can: 
+
+* Deploy APIs to your cluster.
+* Apply policies and other configurations.
+* Create all resources through Anypoint Platform. 
```

Comment:
There are some configuration that must be provided as local files/k8s resources (for example, the `Configuration` resource).
```suggestion
```

Reply:
@seblaz We can delete that inaccurate bullet, but readers will almost certainly want to know what types of resources they can create (and can't created) via Anypoint Platform.

Is there something we can say here about what they can create through Anypoint Platform in connected mode (compared to what they can't create through Anypoint Platform when in local mode).